sdk/monitor/azure-monitor-query/azure/monitor/query/_helpers.py

```diff
@@ -90,3 +98,22 @@ def native_col_type(col_type, value):
 
 def process_row(col_types, row):
     return [native_col_type(col_types[ind], val) for ind, val in enumerate(row)]
+
+def process_error(error):
```

Comment:
Here, error is an HttpResponseError, right?

Reply:
yes that is correct

Reply:
Could you help me understand what is this method for?

Reply:
actually, made some small changes to this - missed a commit earlier
this is to convert the model into the logsqueryerror intead of generated internal model.